src/daemons/rhsmcertd.c

```diff
@@ -410,6 +435,13 @@ parse_cli_args (int *argc, char *argv[])
 int
 main (int argc, char *argv[])
 {
+    debug("Getting seed for random number generator from");
+    // Grab a seed using the getrandom syscall
+    unsigned long int seed;
+    syscall(SYS_getrandom, &seed, sizeof(unsigned long int), 0);
+    srand(seed);
+
+    info("finished random stuff: %d", seed);
```

Comment:
How about `Using seed %d` "random stuff:" followed by an integer isn't very meaningful to external folks.